x-pack/plugins/dashboard_enhanced/public/services/drilldowns/actions/flyout_create_drilldown/flyout_create_drilldown.tsx

```diff
@@ -41,6 +41,15 @@ export class FlyoutCreateDrilldownAction implements ActionByType<typeof OPEN_FLY
     const supportedTriggers = context.embeddable.supportedTriggers();
     if (!supportedTriggers || !supportedTriggers.length) return false;
     if (context.embeddable.getRoot().type !== 'dashboard') return false;
+
+    /**
+     * Temporarily disable drilldowns for Lens as Lens embeddable does not have
+     * `.embeddable` field on VALUE_CLICK_TRIGGER context.
+     *
+     * @todo Remove this condition once Lens adds `.embeddable` to field to context.
+     */
+    if (context.embeddable.type === 'lens') return false;
```

Comment:
@streamich, should we instead check:
```
if (context.embeddable.type === 'visualisation') return true
return false;
```

I am not sure, if this also will be an issue, for example, for ml which will be merged soon. or for other types

Reply:
It should be a feature that drilldowns work with any embeddable.

For lens, they need to provide `{ embeddable }` in trigger and it should work.

For ML, drilldowns are already disabled, they would to implement `.supportedTriggers()` on their embeddable to enable drilldowns.

Reply:
One thing we could do is through embeddable input or embeddable itself specify if drilldowns should be enabled.

```ts
embeddable.drilldownsEnabled
embeddable.getInput().drilldownsEnabled
```